Formula/rpki-client.rb

```diff
@@ -0,0 +1,49 @@
+class RpkiClient < Formula
+  desc "OpenBSD portable rpki-client"
+  homepage "https://github.com/rpki-client/rpki-client-portable"
+  url "https://github.com/rpki-client/rpki-client-portable/archive/refs/tags/7.4.tar.gz"
+  sha256 "cacb56107b29baef67c0117bc53d726c95684ad27a6affe2a1527470c67f5e86"
+  license "ISC"
+
+  depends_on arch: :intel
+  depends_on "autoconf"
+  depends_on "automake"
+  depends_on "libressl"
+  depends_on "libtool"
+  depends_on macos: :big_sur
+  depends_on "pkg-config"
+
+  def install
+    ENV.deparallelize
+
+    # git fetch the OpenBSD head codestate, and apply the patches to convert it to portable
+    system "./autogen.sh"
+
+    # --localstatedir wires things into the installed /var/{db,cache}/rpki-client paths, not the symlink farm
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}",
+                          "--sysconfdir=#{HOMEBREW_PREFIX}/etc/",
+                          "--localstatedir=#{HOMEBREW_PREFIX}/var"
+    system "make", "install"
+  end
+
+  def post_install
+    # make the var/db,cache/rpki-client dirs
+    (var/"db/rpki-client").mkpath
+    (var/"cache/rpki-client").mkpath
+  end
+
+  def caveats
+    <<~EOS
+      If you accept the ARIN Relying Party Agreement (RPA), and want to install the arin.tal:
+
+      curl -o "#{etc}/rpki/arin.tal" https://www.arin.net/resources/manage/rpki/arin.tal
+    EOS
+  end
```

Comment:
```suggestion
```
`caveats` should be Homebrew-specific.

Reply:
I'll find another way to do this kind of user-hint